packages/oas-to-snippet/__tests__/index.test.js

```diff
@@ -279,17 +260,61 @@ test('should not double-encode query strings', () => {
 });
 
 describe('supported languages', () => {
-  const languages = Object.keys(supportedLanguages)
-    .filter(lang => lang !== 'node-simple')
-    .map(lang => [lang]);
+  const languages = Object.keys(supportedLanguages).map(lang => [lang]);
+
+  describe.each(languages)('%s', lang => {
+    const targets = Object.keys(supportedLanguages[lang].httpsnippet.targets);
+
+    it('should have a language definition', () => {
+      expect(supportedLanguages[lang]).toMatchObject({
+        highlight: expect.any(String),
+        httpsnippet: {
+          lang: expect.any(String),
+          default: expect.any(String),
+          targets: expect.any(Object),
+        },
+      });
+
+      expect(targets.length).toBeGreaterThanOrEqual(1);
+      expect(targets).toContain(supportedLanguages[lang].httpsnippet.default);
+
+      targets.forEach(target => {
+        if ('opts' in supportedLanguages[lang].httpsnippet.targets[target]) {
+          // eslint-disable-next-line jest/no-conditional-expect
+          expect(supportedLanguages[lang].httpsnippet.targets[target].opts).toStrictEqual(expect.any(Object));
+        }
+
+        if ('install' in supportedLanguages[lang].httpsnippet.targets[target]) {
+          // eslint-disable-next-line jest/no-conditional-expect
+          expect(supportedLanguages[lang].httpsnippet.targets[target].install).toStrictEqual(expect.any(String));
+        }
```

Comment:
I don't really like this conditional testing, but Jest doesn't offer any "assert this value as something if it exists, if it doesn't its ok". I could do an assertion on this with JSON Schema but that'd require loading in `ajv` or some other validator. No thanks!